app/components/UserAttendance/AttendanceModal.css

```diff
@@ -1,63 +1,135 @@
-.list {
-  width: 100%;
-  min-height: 100px;
-  max-height: calc(100vh - 210px);
-  padding: 0 30px;
-  margin-bottom: 5px;
-  overflow-y: auto;
+.modal {
+  height: calc(100vh - 400px);
 }
 
-.list > li {
-  border-bottom: 1px solid var(--color-mono-gray-3);
+.search {
+  width: 60%;
+  height: 35px;
+  padding: 10px;
+  border: 1px solid var(--color-light-gray-1);
+  border-radius: 5px;
+  color: var(--color-gray-5);
+
+  &:focus-within {
+    border-color: var(--color-gray-5);
+  }
 }
 
-.list > li:last-child {
-  border-bottom: none;
+/* Search icon */
+.search > div {
+  color: var(--color-light-gray-1);
 }
 
-.nav {
-  display: flex;
-  flex: 1;
-  padding: 5px;
+.search input {
+  border: none;
+  outline: none;
+  box-shadow: none;
+  font-size: 0.9rem;
+
+  &:focus {
+    border: none;
+    box-shadow: none;
+  }
+
+  &::placeholder {
+    color: var(--color-light-gray-1);
+  }
+}
+
+html[data-theme='dark'] .search input {
+  background: none;
+}
+
+.list {
+  position: relative;
   width: 100%;
-  justify-content: center;
+  min-height: 100px;
+  height: 80%;
+  overflow-y: auto;
+  /* stylelint-disable indentation */
+  background:
```

Comment:
Kind of proud of this one. It adds a shadow to the top and bottom as you scroll. So, no shadow at the top when at the top and no shadow at the bottom when at the bottom.